diff --git a/src/schema.rs b/schema.rs.save
index 680e868..c6e639a 100644
--- a/src/schema.rs
+++ b/schema.rs.save
@@ -1,7 +1,10 @@
 table! {
+    use diesel::sql_types::*;
+    use crate::models::game::{Bavarian_suit, Game_type, Schneider_schwarz};
+
     games (id, session_id, group_id) {
         id -> Int4,
         session_id -> Int4,
         group_id -> Int4,
         game_type -> Game_type,
         suit -> Nullable<Bavarian_suit>,
@@ -39,12 +42,15 @@ table! {
         abbreviation -> Varchar,
         name -> Varchar,
     }
 }
 
 table! {
+    use diesel::sql_types::*;
+    use crate::models::rule_set::Count_laufende;
+
     rule_sets (id) {
         id -> Int4,
         name -> Varchar,
         count_laufende -> Count_laufende,
         geier_allowed -> Bool,
         hochzeit_allowed -> Bool,
